te/TE.py

```diff
@@ -747,13 +747,11 @@ def __get_cpu_count_tedps(self, te_dp_hosts, client):
             cpu_result = {}
             problematicHost = {}
             for host, linesOfOutput in output.items():
-                for line in linesOfOutput:
-                    try:
-                        cpus = int(line)
-                        cpu_result[host] = cpus
-                    except:
-                        problematicHost[host] = line
-                    break
+                try:
+                    cpus = int(linesOfOutput)
```

Comment:
@Nikesh1177, was this tested, int could break if we have `\n` in `linesOfOutput`